its/pom.xml

```diff
@@ -124,36 +124,4 @@
     </plugins>
   </build>
 
-  <profiles>
-    <profile>
-      <id>download-scanner-artifacts</id>
-      <activation>
-        <property>
-          <name>scannerForMSBuild.version</name>
```

Comment:
When `scannerForMSBUILD.version` parameter is present in the maven command (see `-DscannerForMSBuild.version=...` removal in yml), then run this profile.

And this profile will download one version of the ZIP from maven maven cache / repox. This is not needed, as it's redundant to using normal build artifact.

Reply:
I wonder if this part even worked as intended. Does it really override the scanner used by the ITs? Does it download and unpack the zip to the right location? To me it looks like it doesn't, e.g. this does not look like an artifact download location to me:
https://github.com/SonarSource/sonar-scanner-msbuild/blob/95bfcb1c84b103a2d5d59085c95a7e82388fd722/its/src/test/java/com/sonar/it/scanner/msbuild/utils/ScannerClassifier.java#L31-L32

Reply:
It worked before the migration to ScannerCommand. The current implementation uses the binaries from the `build` directory (due to artifact publish/download in yml). Without the artificial roundtrip via repox